.github/workflows/typecheck.yml

```
@@ -23,39 +23,31 @@ jobs:
-          path: node_modules
-          key: ${{ runner.os }}-typecheck-${{ hashFiles('**/yarn.lock') }}
-          restore-keys: |
-            ${{ runner.os }}-typecheck-
-            ${{ runner.os }}-test-
-            ${{ runner.os }}-build-
+          cache: "pnpm"
 
       - name: Install dependencies
-        if: steps.yarn-cache.outputs.cache-hit != 'true'
-        run: yarn install --frozen-lockfile
+        if: steps.pnpm-cache.outputs.cache-hit != 'true'
```

actionlint (1.7.12)

[error] property "pnpm-cache" is not defined in object type {}

(expression)